### ISSUE TITLE
with_cache default build files

### ISSUE THREAD

**Issue by skjolber** [CircleCI-Public/gradle-orb#29]
The default build files are now [build.gradle, build.gradle.kts, settings.gradle.kts, libs.versions.toml](https://github.com/CircleCI-Public/gradle-orb/blob/main/src/commands/with_cache.yml#L20).

Since `settings.gradle.kts` is included, should not the corresponding `settings.gradle` be included as well?

Also the description is no longer up to date.

**Comment by skjolber** [comment]
Also the description 

> The contents of the `~/.gradle` directory is cached, which will substantially..

should be adjusted to 

> The contents of the `~/.gradle/cache` directory is cached, which will substantially..

 `~/.gradle` contains secrets via global gradle.properties so the docs should reflect that it is in fact not cached.

**Comment by skjolber** [comment]
Also now that there is two `save_cache` and `restore_cache` they should get a name each, i.e. "Restoring gradle wrapper cache" and "Restoring gradle cache" or something like that.

**Comment by ryanbourdais** [comment]
Closed by PR #31

**Comment by skjolber** [comment]
@ryanbourdais got the `restore_cache` descriptions, but what about `save_cache`?

**Comment by ryanbourdais** [comment]
Good catch, I just submitted a follow up PR to add names to those 2 steps as well
